src/rt-api/post-contact.ts

```diff
@@ -25,16 +31,23 @@ export const postContact = async (
   }
 
   // If present, update contact's street addresses
-  if (contact.Address) {
-    contact.Address.forEach(async (addressItem: AddressUpdate) => {
+  if (contact.Addresses) {
+    contact.Addresses.forEach(async (address: AddressUpdate) => {
       try {
+        const method = address.id === 0 ? "post" : "put";
+        const addressUrl =
+          address.id === 0
+            ? `/contacts/${contact.ContactRecord.id}/addresses`
+            : `/contacts/${contact.ContactRecord.id}/addresses/${address.id}`;
+        if (address.id === 0) {
+          delete address.id;
+        }
```

Comment:
This is if statement kind of smells... We're checking if `address.id === 0` three lines in a row. Maybe we can consolidate this into one if block somehow? Thoughts?